tests/cypress/tests-e2e/drawing.cy.js

```diff
@@ -50,7 +50,14 @@ describe('Drawing module tests', () => {
             )
             cy.readStoreValue('state.features.selectedFeatures[0].title').should('eq', title)
         }
-
+        function readCoordinateClipboard(name) {
+            cy.get(`[data-cy="${name}-button"]`).focus()
+            cy.get(`[data-cy="${name}-button"]`).realClick()
+            cy.get(`[data-cy="${name}-icon"]`).should('have.class', 'fa-check')
+            cy.readClipboardValue().then((clipboardText) => {
+                expect(clipboardText).to.be.equal("2'660'013.50, 1'185'172.00")
+            })
+        }
```

Comment:
In cypress I'm not a big fan of such function helper, if for example the second call of the function is failing it is hard (almost impossible) to know form cypress logs which call failed. So either don't use a helper, or add a `cy.log()` in the helper using the input argument to ease the traking of the function call.

Reply:
I would also provide the coordinate as argument, see other comment.